b2b-app/imports/ui/admin/register/form/form-schemas.js

```diff
@@ -108,6 +192,13 @@ const termsFormSchema = new SimpleSchema({
 const contactFormBridge = new SimpleSchema2Bridge(contactFormSchema)
 const aboutFormBridge = new SimpleSchema2Bridge(aboutFormSchema)
 const emergencyFormBridge = new SimpleSchema2Bridge(emergencyFormSchema)
+const avatarFormBridge = new SimpleSchema2Bridge(avatarFormSchema)
 const termsFormBridge = new SimpleSchema2Bridge(termsFormSchema)
 
-export { contactFormBridge, aboutFormBridge, emergencyFormBridge, termsFormBridge }
+export {
```

Comment:
This change, being simply about formatting, suggests that our prettier definitions are not in sync, or we don't have `Format on save` turned on.
It's important that we all have the same settings, because if Andrew edits this file again, it may go back to his layout.

Reply:
I think Prettier moves the exports onto separate lines when the line exceeds a character threshold because I added avatarFormBridge

Reply:
Yeah at some point prettier changes from single line destructuring (and prop passing) to multiline